From 98b67cc76cf774ef46bb391f07593963c7f8aebd Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Tue, 26 Aug 2014 12:49:22 -0400
Subject: [tools] turbostat: Add a .gitignore to ignore the compiled turbostat binary

Message-id: <1409057382-26432-12-git-send-email-prarit@redhat.com>
Patchwork-id: 89087
O-Subject: [RHEL7.1 PATCH BZ 1133893 11/31] turbostat: Add a .gitignore to ignore the compiled turbostat binary
Bugzilla: 1133893
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1133893

commit 7ade7f48b1ffb21387d3694f1666f488c070860d
Author: Josh Triplett <josh@joshtriplett.org>
Date:   Tue Aug 20 17:20:19 2013 -0700

    turbostat: Add a .gitignore to ignore the compiled turbostat binary

    Signed-off-by: Josh Triplett <josh@joshtriplett.org>
    Signed-off-by: Len Brown <len.brown@intel.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/power/x86/turbostat/.gitignore b/tools/power/x86/turbostat/.gitignore
new file mode 100644
index 0000000..7521370
--- /dev/null
+++ b/tools/power/x86/turbostat/.gitignore
@@ -0,0 +1 @@
+turbostat
-- 
1.7.1